src/commands/InitCommand.js

```diff
@@ -4,18 +4,37 @@ import GitUtilities from "../GitUtilities";
 import Command from "../Command";
 import objectAssignSorted from "object-assign-sorted";
 
+const SUPPORTED_OPTS = {
+  exact: {
+    describe: "Specify cross-dependency version numbers exactly rather than with a caret (^)"
```

Comment:
In the context of `InitCommand`, `--exact` is actually intended to pin the version of `lerna` that was used to initialize in `package.json` without a caret. `"Specify lerna dependency version in package.json without a caret (^)"` is probably better, here.